src/sync.js

```diff
@@ -41,6 +41,7 @@ async function processBatch(txs) {
         await addUser(timestamp, payload.work[1].input.worker_account);
         break;
       case "new_account_name":
```

Comment:
I think that I mixed it up and this case should be `account_create` from the beginning.

Reply:
Ho yeah, i will remove this line then.